widgetastic_manageiq/__init__.py

```diff
@@ -1261,17 +1261,31 @@ def get_count(self, item_name):
         """
         return self.primary_card(item_name).card_count()
 
+    @property
+    def is_displayed(self):
+        """Checks if Total service card is displayed"""
+
+        return self.primary_card("Total Services").card.is_displayed
+
 
 class SSUIAggregatecard(Widget, ClickableMixin):
     """Represents an aggregate card like Current Services or Retired services."""
 
     @ParametrizedView.nested
     class aggregate_card(ParametrizedView):  # noqa
         PARAMETERS = ("item_name",)
-        card = Text(ParametrizedLocator('.//div[@class="card-pf-body"]'
-            '/h2[normalize-space(.)={item_name|quote}]'))
-        count = Text(ParametrizedLocator('.//div[@class="card-pf-body"]'
-            '/p[./preceding-sibling::h2[normalize-space(.)={item_name|quote}]]/span[2]'))
+        card = VersionPick({
```

Comment:
the change of this locator makes me wonder if this wouldn't create trouble,

i presume the click event will bubble up to the outer body, and so e the new version will work in both cases, card_click and card_count